frontend/app/analysis/index/template.hbs

```diff
@@ -22,9 +23,6 @@
         @onReset={{queue t.clear this.reset}}
         as |fs|
       >
-        {{#if this.prefetchData.isRunning}}
```

Comment:
Wanna re-add this in a unless/else block or is this not of value?

Reply:
it is of value, without this `@initial` gets set before the the data is prefetched, which then causes this:

![image](https://github.com/user-attachments/assets/a0b97976-502b-41d1-92be-9f38a70511f6)

Reply:
the problem is that `@initial` gets set before the data is fetched, which results in this:

![image](https://github.com/user-attachments/assets/bd227373-31d0-4699-a1e8-f0b4c17de86d)